static_src/components/app_container.jsx

```diff
@@ -166,18 +170,22 @@ export default class AppContainer extends React.Component {
               </PageHeader>
             </div>
           </div>
-          <Panel title="Usage and allocation">
-              <span>View more usage data at <a href="https://logs.cloud.gov">logs.cloud.gov</a></span>
+          <Panel title="App usage and allocation" helperText={ usageHelperText }>
             <UsageLimits app={ this.state.app } quota={ this.state.quota } />
           </Panel>
 
-          <Panel title="Routes">
-            <RoutesPanel />
-          </Panel>
-
-          <Panel title="Services">
-            <ServiceInstancePanel />
-          </Panel>
+          <Row gutters valign="top">
+            <Col flex={ 1 }>
```

Comment:
This is an example of where I think we went from a simple interface; just dropping panels side by side when you need, to a much more complex one; Using `Col` with a `flex` property that I wouldn't understand as a developer unless I went into the `Col` code. What are the `Col`s doing that can just be done with the `Panel`?